src/main/java/seedu/address/logic/commands/patient/RemarkCommand.java

```diff
@@ -1,12 +1,15 @@
-package seedu.address.logic.commands;
+package seedu.address.logic.commands.patient;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
 import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
```

Comment:
Maybe we should have RemarkCommand for both patient and appointment? so we split into RemarkPatientCommand and RemarkAppointmentCommand

Reply:
Appointment class already has a String description, is that good enough?